src/Core/src/Handlers/TimePicker/TimePickerHandler.iOS.cs

```diff
@@ -26,7 +32,7 @@ protected override void ConnectHandler(MauiTimePicker platformView)
 				platformView.EditingDidEnd += OnEnded;
 				platformView.ValueChanged += OnValueChanged;
 				platformView.DateSelected += OnDateSelected;
-				platformView.Picker.ValueChanged += OnValueChanged;
+				platformView.PickerValueChanged += OnValueChanged;
```

Comment:
These are all now weak event handler on the MauiTimePicker. iOS is able to collect this a lot better since it's a weakeventhandler